core/src/test/java/kafka/test/ClusterInstance.java

```diff
@@ -145,4 +152,11 @@ default Admin createAdminClient() {
     void startBroker(int brokerId);
 
     void waitForReadyBrokers() throws InterruptedException;
+
+    default Iterable<GroupProtocol> supportedGroupProtocol() {
+        return this.config().serverProperties()
+                .get(NEW_GROUP_COORDINATOR_ENABLE_CONFIG).equals("true")
```

Comment:
Also, this shared method needs to check `GROUP_COORDINATOR_REBALANCE_PROTOCOLS_CONFIG`. Other tests could use `GROUP_COORDINATOR_REBALANCE_PROTOCOLS_CONFIG` to enable new consumer protocol